readme.md

```diff
@@ -50,7 +50,9 @@ This section contains libraries that are well-made and useful, but have not nece
 - [efax](https://github.com/NeilGirdhar/efax) - Exponential Families in JAX. <img src="https://img.shields.io/github/stars/NeilGirdhar/efax?style=social" align="center">
 - [mpi4jax](https://github.com/PhilipVinc/mpi4jax) - Combine MPI operations with your Jax code on CPUs and GPUs. <img src="https://img.shields.io/github/stars/PhilipVinc/mpi4jax?style=social" align="center">
 - [imax](https://github.com/4rtemi5/imax) - Image augmentations and transformations. <img src="https://img.shields.io/github/stars/4rtemi5/imax?style=social" align="center">
-
+- [T5X](https://github.com/google-research/google-research/tree/master/flax_models/t5x) - T5 Seq2Seq model. 
```

Comment:
This implementation seems like a better fit for the "Models and Projects" section. Also, I think it makes sense to link all of https://github.com/google-research/google-research/tree/master/flax_models instead of just this one model. What do you think?

Reply:
Yes, that's a good idea. Can you make that change?